cmd/manager/main.go

```diff
@@ -106,17 +107,29 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := monitoringv1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "error registering prometheus monitoring objects")
+		os.Exit(1)
```

Comment:
you can use log.Fatalf instead of log.errr and then os.exit

Reply:
I'm using what was above in 106-107 that was generated from operator-sdk